lib/themes/dosomething/paraneue_dosomething/js/components/ContextSlide.js

```diff
@@ -11,9 +11,21 @@ class ContextSlide extends React.Component {
 
   render() {
     return (
-      <div className="carousel__slide">
-        <h1>{"Woohoo! You're signed up."}</h1>
-        <p>here is some text</p>
+      <div className="container slideshow__slide">
```

Comment:
Yeah, I'll switch it to it's own `slideshow__slide` wrapping element. I was just starting to feel a strong case of divitis coming on ;)